### PR TITLE
bower install dependencies. closes #37

### LINKED ISSUE #37: Use package manager to install js dependencies
Right now we commit all of our dependencies. This is bad. We should use something like bower to install js dependencies so we don't have to commit them to the repo

### LINKED ISSUE #40: make example config
config.py is gitignored for good reason but we should make an example one and just share the important keys individually with a template to work from

### PR DESCRIPTION
**Pull request by pwgraham91**
closes https://github.com/pwgraham91/ELO-Darts/issues/37
closes https://github.com/pwgraham91/ELO-Darts/issues/40
Bower install dependencies instead of committing them
Also provides example config

### CODE REVIEW

**Comment by pwgraham91** [comment]
look at that diff!! ahhh yes

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ELO-Darts",
+  "homepage": "https://github.com/pwgraham91/ELO-Darts",
+  "authors": [
+    "Peter Graham <pwgraham91@gmail.com>",
+    "Richard Young <richardy@cratejoy.com>",
```
Comment (pwgraham91) [inline bower.json:6]: @youngrichard let me know if you want your personal email here

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "Zach Royer <zachary@cratejoy.com>"
```
Comment (pwgraham91) [inline bower.json:7]: @zroyer let me know if you want your personal email here

**Comment by zroyer** [comment]
let's add `bower install` to the readme but lgtm!

**Comment by youngrichard** [comment]
👍

**Pull request merged**